### PR TITLE
First Version of Send Meme

### PR DESCRIPTION
**Pull request by anmolm96**
Changes-
* Frontend:
  * Working form page with:
    * Imgur File Upload
    * Google Maps Autocomplete Address
    * Stripe Payment Button
* Backend:
  * Processing payment + create postcard on Lob

Future Work:
* Styling
* Landing Page
* Amount on the Form page
* (possibly) rename ```send-meme``` page
* PH Page + Medium Post

### CODE REVIEW

File: api/index.js [hunk 1]
```diff
@@ -12,11 +12,58 @@
...
+      description: 'Demo Postcard job',
+      to: {
+        name: name,
+        address_line1: street1,
+        address_line2: street2,
+        address_city: city,
+        address_state: state,
+        address_zip: zip
+      },
+      front: 'tmpl_3dd5a5c271e43e1',
```
Comment (anmolm96) [inline api/index.js]: Will change once we style and actually start charging

File: api/index.js [hunk 2]
```diff
@@ -12,11 +12,58 @@
...
+        name: name,
+        from: from,
+        meme_url: meme_url
+      }
+    }, (err, result) => {
+      if(err){
+        res.end("error");
+      }
+      else{
+        res.end("successful");
```
Comment (anmolm96) [inline api/index.js]: We might want to make this more descriptive?

File: app/components/AddressAutoCompleteInput.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+    this.handleSelect();
+    console.log(this.state);
+  }
+
+  render() {
+    return (
+      <div style={mainDivStyle}>
+        <div id="locationField">
+          <input style={style} id="autocomplete" placeholder="Where to send . . ."
```
Comment (anmolm96) [inline app/components/AddressAutoCompleteInput.js]: Probably should rename placeholder too

File: api/index.js [hunk 4]
```diff
@@ -12,11 +12,58 @@
...
+        address_zip: zip
+      },
+      front: 'tmpl_3dd5a5c271e43e1',
+      back: 'tmpl_730ab28adb860e7',
+      merge_variables: {
+        name: name,
+        from: from,
+        meme_url: meme_url
+      }
+    }, (err, result) => {
```
Comment (sarahgemp04) [inline api/index.js]: Yeah maybe return the error, so for notification of complete payment, I can notify the user of the issue. ie Lob or Stripe?

File: api/index.js (hunk 2 above)
Comment (sarahgemp04) [reply api/index.js]: Added comment above

File: api/index.js (hunk 4 above)
Comment (anmolm96) [reply api/index.js]: Yeppp makes sense!
Can you approve this PR.
Will add in the next

**Pull request merged**